<ui version="4.0" >
 <class>AreasSelectorClass</class>
 <widget class="QMainWindow" name="AreasSelectorClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Areas Selector Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QMcWidget" native="1" name="selectorControl_" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionProperties" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew_Project" />
    <addaction name="separator" />
    <addaction name="actionLoad_Project" />
    <addaction name="actionSave_Project" />
    <addaction name="actionLoad_Image" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuTools" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="actionLoad_Project" >
   <property name="text" >
    <string>Load Project...</string>
   </property>
  </action>
  <action name="actionSave_Project" >
   <property name="text" >
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionLoad_Image" >
   <property name="text" >
    <string>Load Image...</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionProperties" >
   <property name="text" >
    <string>Properties...</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="text" >
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="text" >
    <string>Redo</string>
   </property>
  </action>
  <action name="actionNew_Project" >
   <property name="text" >
    <string>New Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QMcWidget</class>
   <extends>QWidget</extends>
   <header>qmcwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
